scripts/globals/abilities/pets/crimson_howl.lua

```diff
@@ -13,8 +13,15 @@ function onAbilityCheck(player, target, ability)
 end;
 
 function onPetAbility(target, pet, skill, summoner)
-	local duration = 30 + summoner:getMod(MOD_SUMMONING);
-	target:addStatusEffect(EFFECT_WARCRY,9,0,duration);
-	skill:setMsg(MSG_BUFF);
-	return EFFECT_WARCRY;
+    local bonusTime = summoner:getSkillLevel(38) - 300; -- summoning skill
+    if (bonusTime < 0) then
+        bonusTime = 0;
+    elseif (bonusTime > 200) then

```

Comment:
There is utils.clamp() for minimum/maximum restricting values letting you write shorter code. :smiley: 

`local bonusTime = utils.cliamp(summoner:getSkillLevel(38) - 300, 0, 200);`
2 lines instead of 6.  (you'd need to require the utils.lua global)
`require("scripts/globals/utils");`